### ISSUE TITLE
convert_caffe fails to find module

### ISSUE THREAD

**Issue by john-f** [mil-tokyo/webdnn#417]
Trying to run the caffe converter gives me this error on ubuntu 16.04 LTS using pyenv

```
python3 ./bin/convert_caffe.py --input_name data --input_shape '(1,3,224,224)' --output_names fc8 --out output ~/something.caffemodel
Traceback (most recent call last):
  File "./bin/convert_caffe.py", line 15, in <module>
    from webdnn import Graph, Shape
  File "/home/john/.pyenv/versions/3.6.2/lib/python3.6/site-packages/webdnn-1.1.0-py3.6.egg/webdnn/__init__.py", line 5, in <module>
    from webdnn import backend
  File "/home/john/.pyenv/versions/3.6.2/lib/python3.6/site-packages/webdnn-1.1.0-py3.6.egg/webdnn/backend/__init__.py", line 2, in <module>
    from webdnn.backend import fallback
  File "/home/john/.pyenv/versions/3.6.2/lib/python3.6/site-packages/webdnn-1.1.0-py3.6.egg/webdnn/backend/fallback/__init__.py", line 1, in <module>
    from webdnn.backend.fallback import generator
  File "/home/john/.pyenv/versions/3.6.2/lib/python3.6/site-packages/webdnn-1.1.0-py3.6.egg/webdnn/backend/fallback/generator.py", line 79, in <module>
    import webdnn.backend.fallback.kernels
  File "/home/john/.pyenv/versions/3.6.2/lib/python3.6/site-packages/webdnn-1.1.0-py3.6.egg/webdnn/backend/fallback/kernels/__init__.py", line 1, in <module>
    from webdnn.backend.fallback.kernels import abs
  File "/home/john/.pyenv/versions/3.6.2/lib/python3.6/site-packages/webdnn-1.1.0-py3.6.egg/webdnn/backend/fallback/kernels/abs.py", line 1, in <module>
    from webdnn.backend.fallback.kernels.elementwise import register_elementwise_kernel
  File "/home/john/.pyenv/versions/3.6.2/lib/python3.6/site-packages/webdnn-1.1.0-py3.6.egg/webdnn/backend/fallback/kernels/elementwise.py", line 7, in <module>
    from webdnn.backend.code_generator.templates.elementwise import generate_elementwise_command_buffer, RegisteredItem
ModuleNotFoundError: No module named 'webdnn.backend.code_generator.templates'

```

I get the same type of error on osx using brew-installed python3

```
python3 ./bin/convert_caffe.py --input_name data --input_shape '(1,3,224,224)' --output_names fc8 --out output ~/something.caffemodel
Traceback (most recent call last):
  File "./bin/convert_caffe.py", line 15, in <module>
    from webdnn import Graph, Shape
  File "/usr/local/lib/python3.6/site-packages/webdnn-1.1.0-py3.6.egg/webdnn/__init__.py", line 5, in <module>
    from webdnn import backend
  File "/usr/local/lib/python3.6/site-packages/webdnn-1.1.0-py3.6.egg/webdnn/backend/__init__.py", line 2, in <module>
    from webdnn.backend import fallback
  File "/usr/local/lib/python3.6/site-packages/webdnn-1.1.0-py3.6.egg/webdnn/backend/fallback/__init__.py", line 1, in <module>
    from webdnn.backend.fallback import generator
  File "/usr/local/lib/python3.6/site-packages/webdnn-1.1.0-py3.6.egg/webdnn/backend/fallback/generator.py", line 79, in <module>
    import webdnn.backend.fallback.kernels
  File "/usr/local/lib/python3.6/site-packages/webdnn-1.1.0-py3.6.egg/webdnn/backend/fallback/kernels/__init__.py", line 1, in <module>
    from webdnn.backend.fallback.kernels import abs
  File "/usr/local/lib/python3.6/site-packages/webdnn-1.1.0-py3.6.egg/webdnn/backend/fallback/kernels/abs.py", line 1, in <module>
    from webdnn.backend.fallback.kernels.elementwise import register_elementwise_kernel
  File "/usr/local/lib/python3.6/site-packages/webdnn-1.1.0-py3.6.egg/webdnn/backend/fallback/kernels/elementwise.py", line 7, in <module>
    from webdnn.backend.code_generator.templates.elementwise import generate_elementwise_command_buffer, RegisteredItem
ModuleNotFoundError: No module named 'webdnn.backend.code_generator.templates'
```

**Comment by Kiikurage** [comment]
I found that `/src/graph_transpiler/webdnn/backend/code_generator/__init__.py` doesn't import `templates` directory and fixed it via #418.
Please reinstall latest revision WebDNN, and try again.

**Comment by john-f** [comment]
Thanks, that was fast! It's fixed :)
